src/resources/database/migrations/2020_10_25_101244_update_permissions_to_appshell_v2.php

```diff
@@ -8,24 +8,32 @@
 class UpdatePermissionsToAppshellV2 extends Migration
 {
     private $permissionsToMigrate = [
-        'list propertyvalues' => 'list property values',
+        'list propertyvalues'   => 'list property values',
```

Comment:
Please revert those style changes, and make sure StyleCI passes

Reply:
Done